routes/apiRoutes.js

```diff
@@ -0,0 +1,69 @@
+const apiRouter = require("express").Router();
```

Comment:
good job importing all external libraries at the top of the file - great for readability!